src/preprocessing/scripts/filter_data.py

```diff
@@ -133,6 +135,8 @@ def reformat_and_filter_dataset(
         filter_fns.append(has_documents_with_min_length())
         filter_fns.append(has_valid_alphanum_fraction())
         map_fns.append(mask_phone_and_email())
+        filter_fns.append(remove_urlj())
+        filter_fns.append(remove_strange())
```

Comment:
関数の種類について1点修正をお願いします！
というのも、URLの削除と謎文字の削除は、どちらも文章を落とす(filter処理)というよりは文章を改変する(map処理)のためです。
```suggestion
        map_fns.append(remove_urlj())
        map_fns.append(remove_strange())
```

Reply:
[c596576](https://github.com/GENIAC-Team-Ozaki/oss-llm-japanese/pull/29/commits/c596576aa954978e1396111fdee45ba38c4a37d3) で修正済み